src/pages/Home.jsx

```diff
@@ -0,0 +1,38 @@
+import Navbar from "@/components/Navbar";
+import NewsList from "@/components/NewsList";
+import Sidebar from "@/components/Sidebar";
+import { isDayState, selectedCategoryState } from "@/store/rootAtoms";
+import { useRecoilState, useRecoilValue } from "recoil";
+import styles from "@/styles/pages/Home.module.scss";
+import { useParams } from "react-router-dom";
+import { useEffect } from "react";
+import IntroMessage from "@/components/IntroMessage";
+
+function Home() {
+  const { category } = useParams();
+  const [selectedCategory, setSelectedCategory] = useRecoilState(
+    selectedCategoryState
+  );
+  const isDay = useRecoilValue(isDayState);
+
+  useEffect(() => {
+    if (category && category !== selectedCategory) {
+      setSelectedCategory(category);
+    }
+    if (!category && selectedCategory !== "Home") {
+      setSelectedCategory("Home");
+    }
+  }, [category]);
+
+  return (
+    <>
+      <Navbar />
+      <main className={`${styles.base} ${!isDay ? styles.night : ""}`}>
+        <Sidebar></Sidebar>
+        <NewsList></NewsList>
```

Comment:
```suggestion
        <Sidebar/>
        <NewsList/>
```
React에서 자식 컴포넌트에 children이 없을 경우, 셀프 클로징을 하는것이 일반적이에요!

Reply:
앗 생각은 하고 있었는데 엔터를 누르면 <></>으로 자동완성이 돼서 놓친 것 같습니다 ! 감사합니다 :)